lib/consts/consts.go

```diff
@@ -28,7 +28,7 @@ import (
 )
 
 // Version contains the current semantic version of k6.
-const Version = "0.32.0"
+const Version = "0.33.0"
```

Comment:
Let's split this into a separate commit.

Reply:
:+1: will do at the end, after addressing any other feedback, to save myself some effort when rebasing :sweat_smile: